packages/amplify-category-function/src/provider-utils/awscloudformation/utils/packageLayer.ts

```diff
@@ -48,7 +48,13 @@ async function zipLayer(context, resource: Resource) {
     zip.pipe(output);
     [...glob.sync(optPath), ...glob.sync(libPath)]
```

Comment:
I think it would be better to do `glob.sync('*', {cwd: optPath})` here instead of the ternary you've got below

Reply:
Not sure why yet, but that breaks things when you have a file in the `opt` folder.

ENOENT: no such file or directory, lstat 'text.txt'

Reply:
okay, well if what you have works, then it's fine